### DISCUSSION TITLE
Blog Atom/RSS Feed

### DISCUSSION THREAD

**Discussion by aadityabhatia** [FalkorDB/FalkorDB#553] [Ideas]
Consider offering a blog feed in any open format to allow users to subscribe with their favorite clients. This increases your reach significantly since users are not limited to opening your webpage to look for updates.

**Comment by flowt-au** [comment]
+1 I am an old-school RSS fan. :-)

**Comment by gkorland** [comment]
@aadityabhatia we're currently running on Google pages but we're in the transition to a more flexible platform, which should allow it too.
